src/scripts/run.sh

```diff
@@ -5,9 +5,10 @@ airflow initdb  # initdb is not destructive, so can be re-run at startup
 python scripts/mkvars.py
 
 # creating an admin and regular users (nessacary when using RABC=True in the airflow.cnf)
-airflow delete_user -u admin
-airflow delete_user -u dataservices
-airflow delete_user -u team_ruimte
+# if user are added for the first time, or credentails are changed, then run delete_user in order to create user 
+#airflow delete_user -u admin
+#airflow delete_user -u dataservices
+#airflow delete_user -u team_ruimte
```

Comment:
Als deze code nooit meer nodig is, gewoon weghalen.

Reply:
Ok check